src/Okta.Idx.Sdk.E2ETests/Steps/MfaWithPasswordAndSmsSteps.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace Okta.Idx.Sdk.E2ETests.Steps
+{
+    [Binding]
+    public class MfaWithPasswordAndSmsSteps : BaseTestSteps
+    {
+        public MfaWithPasswordAndSmsSteps(ITestContext context)
+                : base(context)
+        { }
+
+        [Given(@"a SPA, WEB APP or MOBILE Policy that defines MFA with Password and SMS as required")]
+        [Given(@"an Authenticator Enrollment Policy that has PHONE as optional and EMAIL as required for the Everyone Group")] 
+        public void GivenASPAWEBAPPOrMOBILEPolicyThatDefinesMFAWithPasswordAndSMSAsRequired()
```

Comment:
Are multiple Givens by design?

Reply:
There were two functions, both doing nothing, I merged them to this one. We can delete all those empty functions BTW, but this requires more changes to *.feature files